platform/android/java/lib/src/org/godotengine/godot/input/GodotInputHandler.java

```diff
@@ -157,6 +162,19 @@ public void run() {
 		return true;
 	}
 
+	/**
+	 * Handles mouse drag event
+	 * Mouse drag (mouse pressed and move) doesn't fire onGeneticMotionEvent so this is needed
+	 * */
+	public boolean handleMouseDragEvent(final MotionEvent event) {
+		if (event.getAction() != MotionEvent.ACTION_MOVE) {
+			// we return true because every time a mouse event is fired, the event is already handled
+			// in onGenericMotionEvent, so by touch event we can say that the event is also handled
+			return true;
+		}
+		return handleMouseEvent(event, false);
+	}
+
```

Comment:
Remove this method. The logic here will be handled in `onTouchEvent` as mentioned below.